docs/getting-started/cash-app-pay.md

```diff
@@ -0,0 +1,220 @@
+---
+layout: default
+title: Cash App Pay
+parent: Getting Started
+nav_order: 4
+---
+
+# Cash App Pay
+{: .d-inline-block .no_toc }
+NEW (v4.3.0)
+{: .label .label-green }
+
+
+<details markdown="block">
+  <summary>
+    Table of contents
+  </summary>
+  {: .text-delta }
+- TOC
+{:toc}
+</details>
+
+
+{: .alert }
+> Cash App Pay is currently available in the following region(s): US
+
+Cash App Pay support can be added using your Afterpay merchant account. This requires a token to be generated by a server-to-server call to the Afterpay API Create Checkout endpoint with a parameter of `isCashAppPay` set to `true`. This method requires importing and implementing the Cash App PayKit SDK.
```

Comment:
nit: `PayKit` should be `Pay Kit` where used

Reply:
Addressed in [bf06244](https://github.com/afterpay/sdk-android/pull/185/commits/bf062444eb308c1ffa171aa93951333bc266c7cb)